### PR TITLE
 Add Centos 7 for no sandbox

### PR DESCRIPTION
**Pull request by geant44**
CentOS 7 doesn't support sandbox function

### CODE REVIEW

**Review by tessus — CHANGES_REQUESTED** [review]
you removed the `then`. This script won't run.

File: Joplin_install_and_update.sh [hunk 1]
```diff
@@ -180,12 +180,12 @@ SANDBOXPARAM=""
 if command -v lsb_release &> /dev/null; then
   DISTVER=$(lsb_release -is) && DISTVER=$DISTVER$(lsb_release -rs)
   DISTCODENAME=$(lsb_release -cs)
+  DISMAJOR=$(lsb_release -rs|cut -d. -f1)
```
Comment (tessus) [inline Joplin_install_and_update.sh:183]: This should be 1DISTMAJOR` to keep it consistent.

File: Joplin_install_and_update.sh [hunk 2]
```diff
@@ -180,12 +180,12 @@ SANDBOXPARAM=""
...
   #-----------------------------------------------------
   # Check for "The SUID sandbox helper binary was found, but is not configured correctly" problem.
   # It is present in Debian 1X. A (temporary) patch will be applied at .desktop file
   # Linux Mint 4 Debbie is based on Debian 10 and requires the same param handling.
-  if [ $DISTVER =~ Debian1. ] || [ "$DISTVER" = "Linuxmint4" ] && [ "$DISTCODENAME" = "debbie" ]
-  then
+  if [ $DISTVER =~ Debian1. ] || [ "$DISTVER" = "Linuxmint4" ] && [ "$DISTCODENAME" = "debbie" ] || [ "$DISTVER" = "CentOS"] && [ "$DISMAJOR" = 7 ]
```
Comment (laurent22) [inline Joplin_install_and_update.sh:188]: How about version 5 and 6 or future 8 and 9? Will the script need to be changed for all versions?

**Comment by tessus** [comment]
@geant44 are you ignoring any messages on a general principle? you didn't answer my question in the previous PR either.

**Comment by geant44** [comment]
> @geant44 are you ignoring any messages on a general principle? you didn't answer my question in the previous PR either.

Sorry I didn't see the mail

File: Joplin_install_and_update.sh (hunk 2 above)
Comment (geant44) [reply Joplin_install_and_update.sh:188]: I am not sure, but CentOS8 doesn't need it. reference: this [website](http://serverarekore.blogspot.com/2021/01/centos-8-streamjoplin.html?m=0)

**Comment by tessus** [comment]
No worries.  I just didn't know, since we also could have re-opened the previous PR. There are a few things that we have to clarify to merge this.

File: Joplin_install_and_update.sh (hunk 2 above)
Comment (tessus) [reply Joplin_install_and_update.sh:188]: Ok, in that case we are fine. CentOS 5 and 6 should not be used anyway anymore. And if there's really someone out there who uses CentOS 6, we can add that later.

Comment (tessus) [reply Joplin_install_and_update.sh:188]: As mentioned in the previous comment, please change `DISMAJOR` to `DISTMAJOR`.
And also enclose the 7 in double quotes for the test.

**Comment by tessus** [comment]
Please look at the 3 issues I mentioned and fix them. You also have to resolve the conflict first. Then we can merge.

**Comment by tessus** [comment]
Instead of `( [ "$DISTMAJOR" = "7" ] || [ "$DISTMAJOR" = "6" ] )` you can use `[[ "$DISTMAJOR" =~ 6|7 ]]`

The only thing missing is solving the conflict.

**Comment by tessus** [comment]
@geant44 if you need help with anything, e.g. solving the conflicy, please let me know

**Review by geant44 — COMMENTED** [review]
> Instead of `( [ "$DISTMAJOR" = "7" ] || [ "$DISTMAJOR" = "6" ] )` you can use `[[ "$DISTMAJOR" =~ 6|7 ]]`
> 
> The only thing missing is solving the conflict.

I updated .

**Comment by tessus** [comment]
The conflict is still there. You opened the PR and inthe meantimesomeone change the same line of code you did. Thus you have to merge the changes and solve the conflict.

You only replaced the code as I suggested but did not resolve the conflict. Please let me know if you need help.

**Comment by geant44** [comment]
I force update it, and it can merge

**Comment by tessus** [comment]
Perfect, now all is fine. Cheers.

**Review by tessus — APPROVED** [review]

**Comment by tessus** [comment]
@laurent22 I forgot, can I merge changes to the script? I can't recall. I think to remember that Caleb and I did before.

**Comment by laurent22** [comment]
Sure, feel free to go ahead

**Pull request merged**